.github/workflows/publish.yml

```diff
@@ -15,5 +15,20 @@ jobs:
       - uses: actions/checkout@v3.3.0
       - name: Check that the image builds
         run: docker build . --file Dockerfile
+  validate-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - name: Validate that action points to main branch
+        run: |
+          BRANCH=$(yq '.jobs.review-approvals.steps[0].uses' $FILE_NAME | cut -d "@" -f2)
```

Comment:
Oh! This is brilliant! I'll use that!

Reply:
Hmm I'm confused now with the branches.
I thought we can replace the `yq` expression with `GITHUB_BASE_REF`?

Reply:
`yq '.jobs.review-approvals.steps[0].uses' $FILE_NAME` get [this line](https://github.com/paritytech/review-bot/blob/88feb92f5727739d28593b1194b8db5c730a41ad/.github/workflows/review-bot.yml#L21): `paritytech/review-bot@main`

And then the `cut -d "@" -f2` gets everything after the `@`.

So, if I didn't modify it, it should point to the repo's main branch (`main`) but if I modified it, then it will point to a different branch.

And on the second line, I do the comparison between the main branch and the target branch. If they don't match, I fail the action and alert the user.

Reply:
Ah, I completely misunderstood the purpose of this.
I thought it's supposed to check this part:

<img width="198" alt="Screenshot 2023-08-08 at 10 00 19" src="https://github.com/paritytech/review-bot/assets/12039224/b0bf287b-6e15-4b21-95d9-a02fc3ec2609">

Can you add a short comment about this in the code? The [link to the line](https://github.com/paritytech/review-bot/blob/88feb92f5727739d28593b1194b8db5c730a41ad/.github/workflows/review-bot.yml#L21) explains a lot.